### ISSUE TITLE
Issue Logging Into Google Account on 1 Profile

### ISSUE THREAD

**Issue by jarretmoses** [brave/brave-browser#23012]
## Description 

Recently I have not been able to log into my google account using my main Brave profile on a non private window. When going through the login process via google I get a `Something went wrong` error on their interface and a network error hitting ` https://accounts.google.com/_/signin/challenge`. If I am on the profile and switch to a private window I can log in fine. If I Create a new profile and use a standard window I can login fine so the issue seems to be on my main profile on a standard window. 

## Steps to Reproduce

I don't know how to reproduce this

## Reproduces how often: 

As of right now I cannot login at all. This issue happens every attempt.

## Brave version (info found on brave://version)

`[Version 1.38.119 Chromium: 101.0.4951.67 (Official Build) (arm64)](https://brave.com/latest/)`

## Version/Channel Information:

- Can you reproduce this issue with the current release?
- Can you reproduce this issue with the beta channel?
- Can you reproduce this issue with the dev channel?
- Can you reproduce this issue with the nightly channel?

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by rebron** [comment]
Hi @jarretmoses. Can you update to the latest 1.39.111 and let us know if you're still seeing this issue?

**Comment by jarretmoses** [comment]
@rebron yes it still occurs on the update

**Comment by derekantrican** [comment]
I've been seeing this too for months. Have not been able to successfully complete any prompts that auth with Google. Here is an example for today authing the Stylus extension to access my Google Drive account for backup (however, this is far from the only place this has happened):

![](https://i.imgur.com/mei0qpd.gif)

Often I use a workaround by copying the URL to a different browser (usually Firefox) and completing the auth that way. That works great for websites, but not for extensions.

**Comment by jarretmoses** [comment]
Yup. That’s the same issue I was having. For me it seemed to be getting caused by an extension I had that once deleted things seemingly were working again. Haven’t had the issue so far since

**Comment by TechPro424** [comment]
Whenever I try signing in to my google account (even with shields down) I finish logging in and then get this: 

![image](https://user-images.githubusercontent.com/86620085/188446173-e41e0484-e770-4f86-a01e-afe801fd341d.png)

**Comment by xVemu** [comment]
Any news?
